src/app/rpmostree-builtin-rollback.c

```diff
@@ -90,11 +90,7 @@ rpmostree_builtin_rollback (int             argc,
 
   if (!opt_reboot)
     {
-      const char *sysroot_path;
-
-      sysroot_path = rpmostree_sysroot_get_path (sysroot_proxy);
-
-      /* By request, doing this without dbus */
```

Comment:
Maybe worth keeping the comment here?  Or clarify it even to:

`Explicitly do the diff locally to avoid sending the rpm headers over dbus`

or so?

Reply:
I just linked to the original discussion!